src/database.h

```diff
@@ -28,7 +28,6 @@ inline auto initStorage(const std::string& file_name, int db_version) {
                                                           sqlite_orm::make_column("abv", &Drink::abv),
                                                           sqlite_orm::make_column("ibu", &Drink::ibu),
                                                           sqlite_orm::make_column("_size", &Drink::_size, sqlite_orm::default_value(0.0)),
-                                                          sqlite_orm::make_column("size", &Drink::_size, sqlite_orm::default_value(0.0)),
```

Comment:
The problem might have been the reference to `&Drink::_size` for the `make_column("size...` call.